app/src/main/java/org/fossasia/pslab/adapters/PerformExperimentAdapter.java

```diff
@@ -176,6 +181,8 @@ public Fragment getItem(int position) {
                         return InvertingOpAmpExperiment.newInstance();
                     case "Non Inverting Op-Amp":
                         return NonInvertingOpAmpExperiment.newInstance();
+                    case "RC Phase Shift":
+                        return RCPhaseShiftExperiment.newInstance();
                     case "Precision Rectifier":
```

Comment:
Removed Summing Junction experiment setup call.
Please re-instate it and try not to remove important code in resolving merge conflicts.
Thanks